soc/nordic/nrf54h/cpurad_mgmt.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Nordic Semiconductor nRF54h20 processors family management helper for the radio core.
+ */
+
+#include <zephyr/init.h>
+
+#include <ironside/se/cpuconf.h>
+
+static int nrf54h20_cpurad_init(void)
+{
+	uint32_t dummy = 0xbadcafe;
+
+	return ironside_se_cpuconf_boot_radiocore((uint8_t*)&dummy, sizeof(dummy));
+}
+
+SYS_INIT(nrf54h20_cpurad_init, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);
```

Comment:
use soc hook

Reply:
I want the application system call that starts the network/radio CPU to be as consistent as possible between nrf53 and nrf54h20.

On nrf53 it is done like this:

SYS_INIT(nrf53_cpunet_init, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);

Shall I modify nrf53 to also use soc hook, or is there some reason you would like it different from 53?

Reply:
Yes, `SYS_INIT()` is legacy, all should be moved over to soc hooks, can see an example here: https://github.com/zephyrproject-rtos/zephyr/pull/78429/files likewise boards should use board hooks

Reply:
done